cardinal/world.go

```diff
@@ -334,6 +353,78 @@ func (w *World) StartGame() error {
 	return err
 }
 
+func (w *World) startGameLoop(ctx context.Context, tickStart <-chan time.Time, tickDone chan<- uint64) {
+	w.Logger.Info().Msg("Game loop started")
+	ecslog.World(w.Logger, w, zerolog.InfoLevel)
+	w.emitResourcesWarnings()
+
+	go func() {
+		ok := w.isGameLoopRunning.CompareAndSwap(false, true)
```

Comment:
To reduce the stuff in the World object, this atomic.Bool should be replaced with the single var that tracks the game state/stage. This can happen in a separate PR: https://linear.app/arguslabs/issue/WORLD-820/isgamelooprunning-should-be-removed-in-favor-of-the-stage-tracker

Reply:
will just do as separate pr